src/app/reducers/accounts/redux/accounts.actions.ts

```diff
@@ -37,5 +49,89 @@ export class AccountFilterUpdate implements Action {
 
 }
 
+export class CreateAccount implements Action {
+  readonly type = CREATE_ACCOUNT;
+
+  constructor(public payload: any) {
+  }
+}
+
+export class CreateSuccess implements Action {
+  readonly type = ACCOUNT_CREATE_SUCCESS;
+
+  constructor(public payload: any) {
+  }
+}
+
+export class CreateError implements Action {
+  readonly type = ACCOUNT_CREATE_ERROR;
+
+  constructor(public payload: any) {
+  }
+}
+
+export class UpdateAccounts implements Action {
```

Comment:
Why it's called UpdateAccounts?
let's change name to UpdateAccount, rename type also